src/main/java/core/basesyntax/FIgureSupplier.java

```diff
@@ -0,0 +1,41 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FIgureSupplier {
```

Comment:
The class name 'FIgureSupplier' should start with an uppercase letter following camel case convention, and should not contain any verbs. Consider renaming it to 'FigureSupplier' to follow Java class naming conventions.